operator/v1/types_csi_cluster_driver.go

```diff
@@ -168,6 +168,20 @@ type AWSCSIDriverConfigSpec struct {
 	// +kubebuilder:validation:Pattern:=`^arn:(aws|aws-cn|aws-us-gov|aws-iso|aws-iso-b|aws-iso-e|aws-iso-f):kms:[a-z0-9-]+:[0-9]{12}:(key|alias)\/.*$`
 	// +optional
 	KMSKeyARN string `json:"kmsKeyARN,omitempty"`
+
+	// efsVolumeMetricsRefreshPeriod specifies the frequency, in minutes, at which volume metrics are refreshed.
```

Comment:
Why minutes specifically? Have you considered using a duration and allowing the user to configure between a range?

Reply:
In general we tend to say don't. You have an operator taking this value and plumbing it through to the backend right? Why transfer the complexity to the user, when you can make it easier for them, and provide a nicer interface?

You could make this a duration with custom validation to limit the minimum and maximum (and breakdown of units), or the recommendation would be, if you're certain of minutes, to include the unit in the name, `refreshPeriodMinutes` for example

Reply:
Ack, updated. PTAL